### PR TITLE
Add code coverage for RichTextBox

### PR DESCRIPTION
**Pull request by Zheng-Li01**
Fixes https://github.com/dotnet/winforms/issues/10453

## Proposed changes

- Add unit tests for RichTextBox to test its DrawToBitmap()/LoadFile()/SaveFile()/Paste() methods
-  Add unit tests for RichTextBoxto test its events: DragDrop, DragEnter, DragLeave, DragOver, GiveFeedback,QueryContinueDrag

## Customer Impact

- None

## Regression? 

- No

## Risk

- Minimal

## Test methodology
- Unit tests

 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/11034)

### CODE REVIEW

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/RichTextBoxTests.cs [hunk 1]
```diff
@@ -10673,6 +10673,248 @@ public void RichTextBox_GetCharIndexFromPosition_Invoke_ReturnsExpected(int x, i
...
+        { "" },
+    };
+
+    [WinFormsTheory]
+    [MemberData(nameof(PlainTextData))]
+    public void RichTextBox_Paste_PlainText_Data(string value)
+    {
+        using RichTextBox richTextBox1 = new();
+
+        if (!string.IsNullOrEmpty(value))
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/RichTextBoxTests.cs:10756]: Because of this null-check this test is not doing anything with the last piece of data input (line10747). Do we need another test that sets RichTextBox text to an empty value?
